#! /usr/bin/env python

PACKAGE='skye_controls'
import roslib
roslib.load_manifest(PACKAGE)

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()
#       Name       Type      Level Description     Default Min   Max
gen.add("k_x_",       double_t,    0,    "position gain", 5,     0, 100)
gen.add("k_v_",       double_t,    0,    "velocit gain", 16,     0, 100)
gen.add("k_R_",       double_t,    0,    "attitude gain", 10,     0, 100)
gen.add("k_omega_",   double_t,    0,    "angular velocity gain", 15,     0, 100)
gen.add("k_if_",       double_t,    0,    "integrator gain for force", 0.2,     0, 2)
gen.add("k_im_",       double_t,    0,    "integrator gain for momentum", 0.05,  0, 2)

exit(gen.generate(PACKAGE, "skye_controls", "skye_params"))
